style.json

```diff
@@ -2021,6 +2021,7 @@
       "metadata": {"mapbox:group": "1444849242106.713"},
       "source": "openmaptiles",
       "source-layer": "place",
+      "minzoom": 6,
```

Comment:
Not sure if I like 5 or 6 better here. This appears to affect the administrative division labels. For small to medium size countries, I think z6 works better. But US state labels may be better to display at z5.

Reply:
I think I solved this putting the states into two new layers by rank, check https://github.com/elastic/osm-bright-desaturated-gl-style/pull/6/commits/16049ef52a271f238aa5aaebd2ec4b794d335e8b